### PR TITLE
added implement review change

### LINKED ISSUE #20: Implement Review.tsx
image

Number of props: **2**, _quote, name_
Location: **Home**
React Icon: included

Description: **This component should include a quote icon, a quote, and a name. All text should have font "font-hearts." To test the component, make sure to assign an value to the props when calling the component. (title = "TITLE") This component is call in  src\components\home\Reviews.tsx**

Path: src\components\home\Review.tsx

This example is in Javascript. Check Quin's pr on how to use prop in typescript
[Example](https://github.com/acm-ucr/robosub-website/blob/dev/src/components/Subtitle.jsx)

Check ACM documentation in resource page on how to use react icon.

### PR DESCRIPTION
**Pull request by Illey1**
It might not be perfect please let me know if I need any changes. I used pixels for width because it would not let me use fractions for the w- for some reason. 
image
image

### CODE REVIEW

File: src/components/home/Review.tsx [hunk 1]
```diff
@@ -1,5 +1,20 @@
-const Review = () => {
-  return <div className="">Review</div>;
+import { RiDoubleQuotesL } from "react-icons/ri";
+import React from "react";
+
+const Review = ({ quote, name }: { quote: string; name: string }) => {
+  return (
+    <div className="relative flex w-[500px] rounded-3xl bg-hearts-white pb-4 pl-4 pt-4 shadow-lg shadow-gray-700">
```
Comment (qhgill) [inline src/components/home/Review.tsx]: pt and pb can be combined to py, you should be able to use fractions for width

File: src/components/home/Reviews.tsx [hunk 2]
```diff
@@ -1,11 +1,5 @@
-import Review from "./Review";
-
 const Reviews = () => {
-  return (
-    <div className="">
-      <Review />
```
Comment (qhgill) [inline src/components/home/Reviews.tsx:6]: this should still be here

**Review by qhgill — CHANGES_REQUESTED** [review]
Overall looks good, just some small changes, if width still doesn't work when you use a fraction try looking at the parent component or adjusting the fraction

File: src/components/home/Reviews.tsx (hunk 2 above)
Comment (Illey1) [reply src/components/home/Reviews.tsx:6]: the thing is it gives me an error, so i can't commit the changes. I have to leave them deleted

**Comment by Illey1** [comment]
See, this is what happens when I commit with the <Reviews /> still in the Reviews.tsx file. It gives me an error and prevents me from making a pull request. I checked the the main branch as well, and this is the exact code inside, yet it gives me an error when I try creating a pull request.

**Review by qhgill — APPROVED** [review]

**Pull request merged**